dotnet/src/SemanticKernel.UnitTests/TemplateEngine/CodeTokenizerTests.cs

```diff
@@ -137,7 +138,7 @@ public void ItThrowsWhenSeparatorsAreMissing()
         var template2 = @"call 'f\\'x";
 
         // Act
-        Assert.Throws<TemplateException>(() => this._target.Tokenize(template1));
-        Assert.Throws<TemplateException>(() => this._target.Tokenize(template2));
+        Assert.Throws<SKException>(() => this._target.Tokenize(template1));
```

Comment:
Suggest making this a `[Theory]` with the multiple templates as InlineParameters parameters.